hazelcast/src/test/java/com/hazelcast/multimap/LocalMultiMapStatsTest.java

```diff
@@ -140,7 +142,10 @@ public void testPutAllAndHitsGeneratedMap() {
         Map<Integer, Collection<? extends Integer>> expectedMultiMap = new HashMap<>();
         testPutAllAndHitsGeneratedTemplate(expectedMultiMap,
                 (o) -> {
-                    o.putAllAsync(expectedMultiMap);
+                    // We need to wait for putAllAsync() call to finish
```

Comment:
can you also write here why we had to wait (stats being updated after the put completes)

Reply:
I have used toCompletableFuture() instead of casting. Also added comment about the join() call